packages/runtime/src/schema.ts

```diff
@@ -102,6 +106,20 @@ const connectionArgsWithAccount = {
   },
 }
 
+const SetOptionsInput = new GraphQLInputObjectType({
+  name: 'SetOptionsInput',
+  fields: {
+    syncTimeout: {
+      type: GraphQLInt,
+      description: 'Maximum amount of time to lookup the stream over the network, in seconds',
```

Comment:
it sucks to have this defined in two places.  I'd rather just refer users to the js-ceramic API docs to see the docs on `CreateOpts.syncTimeoutSeconds`

Reply:
Sure, do you have a link to the docs I could use here please?

Reply:
Finally found them: https://developers.ceramic.network/reference/typescript/interfaces/_ceramicnetwork_common.CreateOpts.html

Reply:
Thanks! I added the link